code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/github/GithubUtils.java

```diff
@@ -51,6 +65,10 @@ public String validateAndFetchAccountIdFromToken(String token) {
     @SuppressWarnings("unchecked")
     public Set<ExternalId> externalIds(String token) {
         Map<String, Object> jsonData = getJsonData(token);
+        return externalIds(jsonData);
+    }
+
+    private Set<ExternalId> externalIds(Map<String, Object> jsonData) {
```

Comment:
possibly rename to getExternalIdsFromJson() or internalGetExternalIds().  or possibly rename the line in line 66 as getExternalIdsFromToken() to avoid confusion with the other getExternalIds() below (line 125)